### PR TITLE
feat(azure-iot-digitaltwins-device): Add sendTelemetry method.

### PR DESCRIPTION
**Pull request by anthonyvercolano**
Adds a new method (sendTelemetry) that permits multiple properties to be sent in a single message.

### CODE REVIEW

File: digitaltwins/e2e/digitaltwin_telemetry.js [hunk 1]
```diff
@@ -113,4 +113,60 @@ describe('Digital Twin Telemetry', function () {
...
+    debug('event hubs client: starting...');
+    startEventHubsClient(onEventHubMessage, onEventHubError, startAfterTime, 3000)
+      .then((client) => {
+        debug('event hubs client: started');
+        ehClient = client;
+        debug('registering digital twin client with test interfaceInstance');
+        return digitalTwinClient.register();
+      }).then(function () {
+        debug('digital twin client registered. sending telemetry: ' + testTelemetryBody);
+        return testInterfaceInstance.sendTelemetry(testTelemetryBody);
```
Comment (YoDaMa) [inline digitaltwins/e2e/digitaltwin_telemetry.js]: shouldn't this be sendImplodedTelemetry

File: digitaltwins/device/src/digital_twin_client.ts [hunk 2]
```diff
@@ -385,6 +394,7 @@ export class DigitalTwinClient {
       'iothub-command-statuscode': statusCode argument of the update method
       '$.ifname': interfaceInstances name
       contentType: 'application/json'
+      contentEncoding: 'utf-8'
```
Comment (peolivei2) [inline digitaltwins/device/src/digital_twin_client.ts]: Thanks :)

File: digitaltwins/device/src/digital_twin_client.ts [hunk 3]
```diff
@@ -407,15 +417,57 @@ export class DigitalTwinClient {
...
+  /**
+   * @method                        private _sendImplodedTelemetry
+   * @description                   Sends a telemetry message for a supplied interface.
+   * @param interfaceInstanceName   Name of the instance for this interface.
+   * @param telemetry               The object to be sent.
+   * @param sendCallback (optional) If present, the callback to be invoked on completion of the telemetry,
+   *                                otherwise a promise is returned.
+   */
+  private _sendImplodedTelemetry(interfaceInstanceName: string, telemetry: any, sendCallback: ErrorCallback): void;
+  private _sendImplodedTelemetry(interfaceInstanceName: string, telemetry: any): Promise<void>;
```
Comment (jebrando) [inline digitaltwins/device/src/digital_twin_client.ts]: >sendImplodedTelemetry [](start = 11, length = 21)

After the message is imploded isn't it just a _sendTelemetryMessage

Comment (anthonyvercolano) [reply digitaltwins/device/src/digital_twin_client.ts]: When I remove the old code I'll change this internal name to remove the imploded.
Good comment

**Pull request merged**